cogs5e/models/initiative/effect.py

```diff
@@ -281,11 +283,24 @@ def parse_resist_str(resist_list):
     return ', '.join([str(Resistance.from_dict(r)) for r in resist_list])
 
 
+def parse_stat_choice(arg, _):
+    if not arg in STAT_ABBREVIATIONS:
+        raise InvalidArgument(f"{arg} is not a valid stat")
+    return arg
+
+
+def parse_stat_choice_str(stat):
+    return verbose_stat(stat)
```

Comment:
I think it's safe to just pass `verbose_stat` directly instead of making this wrapper